crates/holochain_cascade/src/test_utils/entry_test_data.rs

```diff
@@ -3,6 +3,7 @@ use holo_hash::ActionHash;
 use holo_hash::EntryHash;
 use holochain_serialized_bytes::UnsafeBytes;
 use holochain_state::prelude::*;
+use holochain_types::prelude::WireNewEntryAction;
```

Comment:
Why is this added?

Reply:
Think I added to the wrong file after clippy complained. Fixed: https://github.com/holochain/holochain/pull/5325/commits/5077dfd70b75223db6640b178b0bf7ac188e59cc